### PR TITLE
Populated initial commit when creating the repository

### PR DESCRIPTION
**Pull request by ad-47**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/versioning/DatasetComponentDAORdbImpl.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ String getPath() {
     }
 
     String getSha256() {
-      return sha256;
+      return sha256 != null ? sha256 : "";
```
Comment (ravishetye) [inline backend/src/main/java/ai/verta/modeldb/versioning/DatasetComponentDAORdbImpl.java]: why is this required? do not set the sha if it is null.

Comment (ravishetye) [reply backend/src/main/java/ai/verta/modeldb/versioning/DatasetComponentDAORdbImpl.java]: I am fine having a function computing the init commit sha , than relaxing a general constraint

Comment (ad-47) [reply backend/src/main/java/ai/verta/modeldb/versioning/DatasetComponentDAORdbImpl.java]: removed and fix it by adding https://github.com/VertaAI/modeldb/pull/215/commits/9549c638abd54f3f121f23ac06f9f9dad8f4c6e3#diff-21f865857eba5394657c27849c19263fR125
